int64 sequence_id
int64 sequence_length
int64 size
CotsDetectionWithoutImage[] detection
CotsMaximumScore[] maximum_scores
